xml/Menu/civiawards.xml

```diff
@@ -7,10 +7,10 @@
     <access_arguments>access CiviCRM</access_arguments>
   </item>
   <item>
-    <path>civicrm/award-review-ssp</path>
+    <path>civicrm/ssp/awardreview</path>
```

Comment:
Same as above.

Reply:
This is fine as it's blends with URL for civicrm